app/experimenter/experiments/admin/nimbus.py

```diff
@@ -13,6 +14,12 @@
 )
 
 
+@admin.action(description="Force jetstream data fetch.")
+def force_fetch_jetstream_data(modeladmin, request, queryset):
+    for experiment in queryset:
+        tasks.fetch_experiment_data(experiment.id)
```

Comment:
This will run the task right in the request handler.  You'll want to `.delay(experiment.id)` here